### ISSUE TITLE
State estimation on Valkyrie

### ISSUE THREAD

**Issue by raluca-scona** [openhumanoids/oh-distro#101]
We would like to have a magnetometer field within the val_hardware_msgs/valImuSensor message type.

**Comment by mauricefallon** [comment]
I made progress on modifying pronto to be just an AHRS i.e. imu-in-orientation-out
https://www.dropbox.com/s/26djpvc3kgnnuon/2016-03-pronto-at-ahrs.mp4?dl=0

Gold is the orientation estimate from the IMU  - swinging around like crazy.
White is the orientation estimate from pronto. The position estimate is fixed at zero.

**Comment by mauricefallon** [comment]
Last week we switched the IMU model from the Microstrain 25 to the 15 and incorporated a bias estimation module into IHMC's estimator. As a result the state estimate by IHMC is quite good. 

@raluca-scona  has been working on removing the need for a 3rd party orientation (this was from POSE_BDI previously). Work on this is in:
https://github.com/openhumanoids/pronto/compare/raluca_pronto_remove_bdi_dependency
This is going to be really neat because it will mean only 3 messages will be needed for the estimator.

I'm sensitive about bringing in this in without testing it on Atlas or closed loop. @patmarion : do you have logs of the Atlas operating recently that we can use?

**Comment by patmarion** [comment]
Yes, but I can pull some recent runs from the onboard computers and upload to virgo
